bundle/manifests/clusterlogging.clusterserviceversion.yaml

```diff
@@ -434,8 +434,6 @@ spec:
                     - ALL
               nodeSelector:
                 kubernetes.io/os: linux
-              securityContext:
```

Comment:
Even though RunAsUser is defined in our custom scc, we should still be adding the runAsNonRoot:true to our manifests.

Reply:
Our custom SCC applies to the collector because those pods mount the host file system of the node.  This change is directed towards the operator.  Why does it need to run as root?